R/rj.b.R

```diff
@@ -64,17 +64,24 @@ RjClass <- R6::R6Class(
 
                 if (Sys.info()['sysname'] == 'Windows') {
 
-                    pathA = file.path(rappdirs::user_data_dir(), 'R', 'win-library', '%v')
-                    pathB = file.path(Sys.getenv('USERPROFILE'), 'R', 'win-library', '%v')
-                    libPaths = paste(pathA, pathB, sep=';')
+                    rcv <- package_version(R$rcv)
+                    if( rcv > package_version('4.1.3') ) {
+                        libPaths=file.path(Sys.getenv('USERPROFILE'), 'AppData', 'Local', 'R', 'win-library', '%v')
+                    } else {
+                        libPaths=file.path(Sys.getenv('USERPROFILE'), 'Documents', 'R', 'win-library', '%v')
+                    }
```

Comment:
> you're confident that R 4.2 always uses appdata/local, and 4.1 always uses documents?
I can't "Put my hand on fire" (Gaio Muzio Scevola 508 BC), because the OS is windows.
I can say that "I have dirty some PCs (with OS win10)" that I had in the availability, with various installations and uninstallations of R (from` 4.1.2` to `4.2.1`), to be able to trace them in the path and in the registry.
As a result, `Documents` for `AppData` is dropped from v4.2.0 for the library.